core/linux-raspberrypi/PKGBUILD

```diff
@@ -5,30 +5,30 @@
 buildarch=20
 
 pkgbase=linux-raspberrypi
-_commit=31f8c0c3f269675085e1271c9d3f76c42e672594
+_commit=dc2d7a137d4ad5f75a1a9a8d82393a88c29c3ba3
 _srcname=linux-${_commit}
 _kernelname=${pkgbase#linux}
 _desc="Raspberry Pi"
-pkgver=4.9.51
+pkgver=4.9.52
 pkgrel=1
 arch=('armv6h' 'armv7h')
 url="http://www.kernel.org/"
 license=('GPL2')
 makedepends=('xmlto' 'docbook-xsl' 'kmod' 'inetutils' 'bc' 'git')
 options=('!strip')
-source=("https://github.com/raspberrypi/linux/archive/${_commit}.tar.gz"
+source=("https://github.com/raspberrypi/linux/archive/${_commit}.tar"
```

Comment:
No.

Reply:
My bad... the RPi takes forever to decompress so did it first but forgot to restore the original.